easybuild/easyconfigs/s/SQLite/SQLite-3.29.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,43 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the
+# policy:  http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SQLite'
+version = '3.29.0'
+
+homepage = 'https://www.sqlite.org/'
+
+description = 'SQLite: SQL Database Engine in a C Library'
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+version_str = '%%(version_major)s%s00' % ''.join('%02d' % int(x) for x in version.split('.')[1:])
+source_urls = ['http://www.sqlite.org/2019/']
+sources = ['sqlite-autoconf-%s.tar.gz' % version_str]
+checksums = ['8e7c1e2950b5b04c5944a981cb31fffbf9d2ddda939d536838ebc854481afd5b']
+
+builddependencies = [('binutils', '2.32')]
```

Comment:
I've rather lost track of all the changing policies here, but assume this is okay.

Reply:
I've had others request that I put single item lists on one line before, so I went and did this for a whole bunch (of which SQLite was one of them). I don't really have a strong opinion myself.

Reply:
Ah, didn't mean the syntax. It was more the use of `GCCcore` with the `builddependency` on `binutils`. Haven't been able to attend too many of the conference calls lately, so just wanted to check this was okay.

Reply:
I see. i targeted the suggested binutils for 2019b; #8567